### PR TITLE
Adding conda_forge_recipe to develop branch

### PR DESCRIPTION
**Pull request by daniloebbinghaus**
Reference to issue https://github.com/AequilibraE/aequilibrae/issues/264.

### CODE REVIEW

File: conda_forge_recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python >= 3.8
+    - cython
+  run:
+    - numpy >= 1.14.0
```
Comment (pedrocamargo) [inline conda_forge_recipe.yaml:25]: Wrong NumPy version

File: conda_forge_recipe.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{% set name = "aequilibrae" %}
+{% set version = "0.7.3" %}
```
Comment (pedrocamargo) [inline conda_forge_recipe.yaml:2]: The develop version is 0.7.4

**Review by janzill — APPROVED** [review]
Nice! For macos this might need to include a link to XCode - see for example [larch's recipe](https://github.com/jpn--/larch/blob/85356fe84dd2f1085a07d7adf763a5d4f163a3d8/conda-recipe/meta.yaml#L13). See also Jeff's comment in [larch's setup file](https://github.com/jpn--/larch/blob/85356fe84dd2f1085a07d7adf763a5d4f163a3d8/setup.py#L10). Approving so this can progress.

**Comment by pedrocamargo** [comment]
@daniloebbinghaus , please follow-up on Jan's comments

**Pull request closed without merging**